.github/workflows/ci.yml

```diff
@@ -75,21 +57,19 @@ jobs:
 
   # Server suite needs to run on VMs not docker
   integration-vagrant:
-    needs: [delivery, yamllint, mdl]
+    needs: lint-unit
     runs-on: macos-10.15
```

Comment:
We should start bumping these up to either latest, 11, or 12 as 10.15 is deprecated https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Reply:
The reason this is still on 10.15 is because the macos-11 image didn't include virtualbox. It looks like they now have it in macos-12 so I'll give that at try.